environment.yml

```diff
@@ -16,60 +16,48 @@ dependencies:
 - cligj=0.4.0=py27_0
 - curl>=7.49.1
 - cycler=0.10.0=py27_0
-- descartes=1.1.0=py27_0
 - enum34=1.1.6=py27_1
 - ephem=3.7.6.0=py27_1
 - expat=2.2.5
-- fiona=1.7.11
 - freetype=2.7=vc9_1
 - freexl=1.0.2=vc9_2
 - functools32=3.2.3.2=py27_1
 - geos=3.6.2
-- gdal=2.2.4
-- geopandas=0.3.0
+- geopandas
```

Comment:
@daren-thomas I prefer we install the exact version of every library to avoid compatibility issues.